R/scregclust.R

```diff
@@ -1487,18 +1577,23 @@ scregclust <- function(expression,
             )
           ))
         }
-
-        r2_test <- 1 - (
-          sum_squares_test / colSums(z2_target_centered^2)
+        # Reset r2_test to be compatible with different active targets
+        r2_test[,] <- 0
```

Comment:
This is a nit-picky style choice, but `,` in square brackets should always be followed by a space, even if the argument after the comma is empty. This may look unusual for you but keeps your additions consistent with the coding style of the rest of the package.